mysql-test/suite/innodb/t/bug84563.test

```diff
@@ -0,0 +1,37 @@
+--source include/count_sessions.inc
```

Comment:
As per our conventions it's better to put lightweight `--source include/have_xxx.inc` includes before other includes which do non-trivial logic.
This is a minor comment which can be ignored

Reply:
fixed